frontend/appflowy_flutter/lib/workspace/presentation/home/panes/flowy_pane.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:appflowy/workspace/application/home/home_setting_bloc.dart';
+import 'package:appflowy/workspace/application/panes/panes.dart';
+import 'package:appflowy/workspace/application/tabs/tabs.dart';
+import 'package:appflowy/workspace/presentation/home/home_layout.dart';
+import 'package:appflowy/workspace/presentation/home/home_stack.dart';
+import 'package:appflowy/workspace/presentation/home/tabs/tabs_manager.dart';
+import 'package:flowy_infra/size.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:provider/provider.dart';
+import 'package:styled_widget/styled_widget.dart';
+
+class FlowyPane extends StatelessWidget {
+  final PaneNode node;
+  final HomeLayout layout;
+  final HomeStackDelegate delegate;
+  const FlowyPane({
+    super.key,
+    required this.node,
+    required this.layout,
+    required this.delegate,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final pageController = PageController();
+    return ChangeNotifierProvider<Tabs>(
+      create: (context) => node.tabs,
+      child: Consumer<Tabs>(
+        builder: (context, value, child) {
+          final horizontalController = ScrollController();
+          final verticalController = ScrollController();
+          return BlocBuilder<HomeSettingBloc, HomeSettingState>(
+            builder: (context, state) {
+              return Scrollbar(
+                controller: verticalController,
```

Comment:
Can we make sure the scrollbar fits the style of the other existing scrollbars, see eg. the vertical scrollbar in the kanban board.

Reply:
Sure thing